io.catenax.shopfloor_information.get_production_tracking/RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this model will be documented in this file.
+
+## [1.0.0] 29.01.2024
```

Comment:
you can change the date to 5th Feb if you want MS3 today.

Reply:
corrections are made. Just as an addition information. Both modesl, this and #556  use the serial_part model version 3.0.0, which is not merged yet.